software/e-label.yml

```
@@ -0,0 +1,17 @@
+name: "E-Label"
+website_url: "https://filipecarneiro.github.io/ELabel/"
+source_code_url: "https://github.com/filipecarneiro/ELabel"
+description: "Open-source solution for electronic labels, with QR Codes, on wine bottles sold within the European Union. This initiative seeks to enhance transparency, efficiency, and information accessibility in the wine industry."
```

The description should only describe what the software does. Additionally terms such as open-source should be avoided ([see here](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines))
Possible alternative description; however, feel free to re-write it yourself according to your preferences!
```suggestion
description: "Solution for electronic labels, with QR code support on wine bottles sold within the European Union."
```